src/Spd.Manager.Licence/BizLicAppBaseValidation.cs

```diff
@@ -14,19 +14,39 @@ public BizLicAppBaseValidator()
         RuleFor(r => r.UseDogs)
             .NotEmpty()
             .When(r => r.CategoryCodes.Contains(WorkerCategoryTypeCode.SecurityGuard));
-        RuleFor(r => r.ApplicantContactInfo.GivenName)
-            .NotEmpty()
-            .When(r => r.ApplicantContactInfo != null);
-        RuleFor(r => r.ApplicantContactInfo.Surname)
-            .NotEmpty()
-            .When(r => r.ApplicantContactInfo != null);
-        RuleFor(r => r.ApplicantContactInfo.PhoneNumber)
-            .NotEmpty()
-            .When(r => r.ApplicantContactInfo != null);
-        RuleFor(r => r.ApplicantContactInfo.EmailAddress)
-            .NotEmpty()
-            .EmailAddress()
-            .When(r => r.ApplicantContactInfo != null);
+        if (typeof(T) == typeof(BizLicAppSubmitRequest))
```

Comment:
why do we need to do this?

Reply:
because the `ApplicantContactInfo ` validation should only be applied when `applicantIsBizManager `is false, in submit request.
the `applicantIsBizManager `is only a member of `BizLicAppSubmitRequest`.

we can also do `ApplicantContactInfo ` validation separately in `BizLicAppUpsertRequestValidator` (with `BizLicAppUpsertRequest`) and `BizLicAppSubmitRequestValidator` (with `BizLicAppSubmitRequest`)